src/app/components/notice-list/notice-list.component.ts

```diff
@@ -43,7 +43,7 @@ export class NoticeListComponent implements OnInit {
   }
 
   ngOnInit() {
-    this.noticeService.noticeUpdate.subscribe( notices => {
+    this.noticeService.notices$.subscribe( notices => {
```

Comment:
Remove subscribe and use async pipe
Enable OnPush
Remove any logic from component to service